### PR TITLE
Adding Automorphic and Armstrong Number Algo

### LINKED ISSUE #1136: Armstrong Number and Automorphic Number in Kotlin
Please Assign it to me.

### PR DESCRIPTION
**Pull request by shubhammantri1**
Fixes #1136  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Adding Automorphic Number Algorithm in Kotlin
- Adding Armstrong Number Algorithm in Kotlin

#### Languages Used:
- Kotlin

#### Files Added:
- Automorphic_Number.kt
- Armstrong_Number.kt

### CODE REVIEW

**Comment by shubhammantri1** [comment]
Please Review it.

File: Armstrong_Number/Armstrong_Number.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    return count
+}
+
+//Function to check Armstrong Number
+fun armstrong(Num: Int, digits: Int):Int {
+    var sum = 0
+    var num = Num
+    while(num > 0) {
+        var digit =  num%10
```
Comment (somya-kapoor) [inline Armstrong_Number/Armstrong_Number.kt]: Add space around operator

Comment (shubhammantri1) [reply Armstrong_Number/Armstrong_Number.kt]: Done

File: Armstrong_Number/Armstrong_Number.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    return count
+}
+
+//Function to check Armstrong Number
+fun armstrong(Num: Int, digits: Int):Int {
+    var sum = 0
+    var num = Num
+    while(num > 0) {
+        var digit =  num % 10
```
Comment (MastersAbh) [inline Armstrong_Number/Armstrong_Number.kt]: you have an extra space after =. Please remove it.

File: Armstrong_Number/Armstrong_Number.kt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    val armNum = armstrong(number, digits)
+    if(armNum == number) {
+        println("Number is an Armstrong Number")
+    }
+    else {
+        println("Number is Not an Armstrong Number")
+    }
+}
+
+//Output:- Armstrong Number
```
Comment (MastersAbh) [inline Armstrong_Number/Armstrong_Number.kt]: A blank line at the end of the file.

File: Automorphic_Number/Automorphic_Number.kt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+fun main() {
+    //Let's take a number 36
+    var num = 76
+    var sqre = num*num
```
Comment (MastersAbh) [inline Automorphic_Number/Automorphic_Number.kt]: space around *

File: Automorphic_Number/Automorphic_Number.kt [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    }
+    if(count >= 1) {
+        println("Not an Automorpic Number")
+    }
+    else {
+        println("Automorpic Number")
+    }
+}
+
+//Output:- Automorphic Number
```
Comment (MastersAbh) [inline Automorphic_Number/Automorphic_Number.kt]: A blank line at the end of the file.

File: Armstrong_Number/Armstrong_Number.kt (hunk 2 above)
Comment (shubhammantri1) [reply Armstrong_Number/Armstrong_Number.kt]: Done

File: Armstrong_Number/Armstrong_Number.kt (hunk 3 above)
Comment (shubhammantri1) [reply Armstrong_Number/Armstrong_Number.kt]: Done

File: Automorphic_Number/Automorphic_Number.kt (hunk 4 above)
Comment (shubhammantri1) [reply Automorphic_Number/Automorphic_Number.kt]: Done

File: Automorphic_Number/Automorphic_Number.kt (hunk 5 above)
Comment (shubhammantri1) [reply Automorphic_Number/Automorphic_Number.kt]: Done

**Comment by jainaman224** [comment]
@MastersAbh @somya-kapoor Please review.

**Pull request merged**